satellite/liveaccounting/liveaccounting.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package liveaccounting
+
+import (
+	"context"
+	"strings"
+	"sync"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+)
+
+// Config contains configurable values for the liveaccounting service.
+type Config struct {
+	StorageBackend string `help:"What to use for storing real-time accounting data"`
+}
+
+// Service represents the external interface to the liveaccounting
+// functionality.
+type Service interface {
+	GetProjectStorageUsage(ctx context.Context, projectID uuid.UUID) (int64, int64, error)
+	AddSpaceUsed(projectID uuid.UUID, inlineSpaceUsed, remoteSpaceUsed int64) error
```

Comment:
seems like we are flipping between `StorageUsage` and `SpaceUsage`, I'm wondering if we want to pick one and stick with it?

Reply:
`StorageUsage` and `SpaceUsed`, actually, yeah. "Storage", to me, is the more general term; it can refer to anything that is "stored" by the system in any way, or to the act or concept of storing. In this particular case, it refers to information about multiple different types of space used. "Space", on the other hand, refers to a specific measurement: the total number of bytes that exist in the system, bytes used by a file, unused bytes on a hard disk, or—in this case—the number of bytes stored inline in the pointerDB and the number of bytes stored remotely.

Reply:
If most people don't see any such distinction between the terms, then yeah, I should change it.

Reply:
that distinction makes sense.  then with your definition,  it seems like they all should be named `SpaceUsage`. I don't have a preference one way or the other, I just like the naming to be consistent so its easier to read.

Reply:
Do you mean the method names or the variable names, or both? I have already changed the method name on line 25 to be `AddProjectStorageUsage()`, and everything else seems to be named according to my definitions there.

Reply:
i think its fine the way it is now.